update.php

```diff
@@ -373,7 +386,10 @@
 		} elseif ( $image === 'cli' ) {
 			// Replace tags inside the WP-CLI Dockerfile template.
 			if ( $config ) {
-				$dockerfile = preg_replace( '|\n%%OLD_PHP%%.*%%/OLD_PHP%%\n|s', '', $dockerfile );
+				if ( '5.2' !== $version ) {
+					$dockerfile = preg_replace( '|\n%%OLD_PHP%%.*%%/OLD_PHP%%\n|s', '', $dockerfile );
+				}
+
 				$dockerfile = str_replace( '%%MYSQL_CLIENT%%', $config['mysql_client'], $dockerfile );
 				$dockerfile = str_replace( '%%DOWNLOAD_URL%%', $config['download_url'], $dockerfile );
 			} else {
```

Comment:
Along with removing `NEW_PHP` from the Dockerfile template, this entire `else` block can go.

Reply:
Left this for now in case we need to add 5.2 specific logic.